sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/MLIRExporter.cs

```diff
@@ -100,10 +106,14 @@ private void ExportBlock(Block block, bool isEntryBlock, MethodDeclarationSyntax
                     }*/
                     break;
                 case SimpleBlock sb:
-                    writer.WriteLine("return");
+                    writer.WriteLine($"br ^{BlockId(sb.SuccessorBlock)}");
                     break;
                 case ExitBlock eb:
-                    writer.WriteLine("return");
+                    if (HasNoReturn(parentMethod))
```

Comment:
If my understanding is correct, considering previous line:
`if (block is ExitBlock && !HasNoReturn(parentMethod))` -> return
`HasNoReturn` should always return true here